tests/integrated/test-squash/runtest

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import netCDF4 as nc
+import itertools as it
+import os
+
+
+def run(cmd):
+    s = os.system(cmd)
+    if s:
+        raise RuntimeError("%s exited with non-zero %d value" % (cmd, s))
+
+
+def verify(f1, f2):
+    org = nc.Dataset(f1)
+    para = nc.Dataset(f2)
+    for v in org.variables:
+        if org[v].shape != para[v].shape:
+            print(org[v])
+            print(para[v])
+            raise RuntimeError("shape mismatch in ", v)
```

Comment:
Put the bad values straight into the exception, rather than printing them

Reply:
how do I get newlines?
Now the error looks terrible:
```
RuntimeError: ('data mismatch in ', 'f2', '(3, 0, 1):-3.0!=-3.0000000000000004\n(3, 0, 2):-3.0!=-3.0000000000000004\n(3, 1, 1):3.0!=3.0000000000000004\n(3, 1, 2):3.0!=3.0000000000000004\n(3, 2, 1):3.0!=3.0000000000000004\n(3, 2, 2):3.0!=3.0000000000000004\n(3, 3, 1):-3.0!=-3.0000000000000004\n(3, 3, 2):-3.0!=-3.0000000000000004\n', BoutArray([[[ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.]],

           [[ 0., -1., -1.,  0.],
            [ 0.,  1.,  1.,  0.],
            [ 0.,  1.,  1.,  0.],
            [ 0., -1., -1.,  0.]],

           [[ 0., -2., -2.,  0.],
            [ 0.,  2.,  2.,  0.],
            [ 0.,  2.,  2.,  0.],
            [ 0., -2., -2.,  0.]],

           [[ 0., -3., -3.,  0.],
            [ 0.,  3.,  3.,  0.],
            [ 0.,  3.,  3.,  0.],
            [ 0., -3., -3.,  0.]],

           [[ 0., -4., -4.,  0.],
            [ 0.,  4.,  4.,  0.],
            [ 0.,  4.,  4.,  0.],
            [ 0., -4., -4.,  0.]]]), BoutArray([[[ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.],
            [ 0.,  0.,  0.,  0.]],

           [[ 0., -1., -1.,  0.],
            [ 0.,  1.,  1.,  0.],
            [ 0.,  1.,  1.,  0.],
            [ 0., -1., -1.,  0.]],

           [[ 0., -2., -2.,  0.],
            [ 0.,  2.,  2.,  0.],
            [ 0.,  2.,  2.,  0.],
            [ 0., -2., -2.,  0.]],

           [[ 0., -3., -3.,  0.],
            [ 0.,  3.,  3.,  0.],
            [ 0.,  3.,  3.,  0.],
            [ 0., -3., -3.,  0.]],

           [[ 0., -4., -4.,  0.],
            [ 0.,  4.,  4.,  0.],
            [ 0.,  4.,  4.,  0.],
            [ 0., -4., -4.,  0.]]]))
```
Its not obvious anymore that boundary cells are affected ...